cloud: add doc for Chat2Query (#11959)
pingcap/docs: tidb-cloud/tidb-cloud-quickstart.md

    The cluster creation process starts and your Serverless Tier cluster will be created in approximately 30 seconds.

## Step 2. Try TiDB Cloud AI capacity

TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL editor in the TiDB Cloud console, to maximize your data value.

>  In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal.

Author: ```suggestion
In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal.
```

    The cluster creation process starts and your Serverless Tier cluster will be created in approximately 30 seconds.

> ## Step 2. Try AI-powered Chat2Query

Author: ```suggestion
## Step 2. Try AI-powered Chat2Query (beta)
```

TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL editor in the TiDB Cloud console, to maximize your data value.

> In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal.

Author: ```suggestion
In Chat2Query, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal.
```

> 1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click your cluster name, and then click **Chat2Query** in the left navigation pane.

Reviewer: ```suggestion
1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click your cluster name, and then click <MDSvgIcon name="icon-left-sql-editor" /> **Chat2Query** in the left navigation pane.
```

2. To try TiDB Cloud AI capacity, follow the on-screen instructions to allow PingCAP and OpenAI to use your code snippets to research and improve the services, and then click **Save and Get Started**.

> 3. In the editor, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually.

Author: ```suggestion
3. In the editor, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually.
```

    > **Note:**
    >

>     > SQL queries generated by AI is not 100% accurate and might still need your further tweak.

Reviewer: ```suggestion
    > SQL queries generated by AI are not 100% accurate and might still need your further tweak.
```

4. Run SQL queries.

   <SimpleTab>

>     <div label="Mac">

Reviewer: ```suggestion
    <div label="macOS">
```

>     For Mac:

Author: ```suggestion
    For macOS:
```

    For Windows or Linux:

    - If you have only one query in the editor, to run it, press **Ctrl + Enter** or click <svg width="1rem" height="1rem" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M6.70001 20.7756C6.01949 20.3926 6.00029 19.5259 6.00034 19.0422L6.00034 12.1205L6 5.33028C6 4.75247 6.00052 3.92317 6.38613 3.44138C6.83044 2.88625 7.62614 2.98501 7.95335 3.05489C8.05144 3.07584 8.14194 3.12086 8.22438 3.17798L19.2865 10.8426C19.2955 10.8489 19.304 10.8549 19.3126 10.8617C19.4069 10.9362 20 11.4314 20 12.1205C20 12.7913 19.438 13.2784 19.3212 13.3725C19.307 13.3839 19.2983 13.3902 19.2831 13.4002C18.8096 13.7133 8.57995 20.4771 8.10002 20.7756C7.60871 21.0812 7.22013 21.0683 6.70001 20.7756Z" fill="currentColor"></path></svg>**Run**.

    - If you have multiple queries in the editor, to run one or several of them sequentially, select the lines of the target queries with your cursor, and then press **Ctrl + Enter** or click **Run**.

    - To run all queries in the editor sequentially, press **Shift + Ctrl + Enter**, or select the lines of all queries with your cursor and click **Run**.

    </div>
    </SimpleTab>

> After running the queries, you can see the query log and results immediately at the bottom of the page.

Reviewer: ```suggestion
After running the queries, you can see the query logs and results immediately at the bottom of the page.
```

## What's next

> - For how to connect to your cluster via a SQL client, see [Connect to a TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster.md).

Reviewer: ```suggestion
- For how to connect to your cluster via different methods, see [Connect to a TiDB cluster](/tidb-cloud/connect-to-tidb-cluster.md).
```

- For more information about how to use Chat2Query to explore your data, see [Chat2Query](/tidb-cloud/explore-data-with-chat2query.md).
- For TiDB SQL usage, see [Explore SQL with TiDB](/basic-sql-operations.md).

> - For production use with the benefits of cross-zone high availability, horizontal scaling, and [HTAP](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing), see [Create a TiDB Cluster](/tidb-cloud/create-tidb-cluster.md) to create a Dedicated Tier cluster.

Reviewer: ```suggestion
- For production use with the benefits of cross-zone high availability, horizontal scaling, and [HTAP](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing), see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md) to create a Dedicated Tier cluster.
```
